docs/book/src/user/quick-start.md

```diff
@@ -80,7 +78,7 @@ Check the [vSphere provider releases](https://github.com/kubernetes-sigs/cluster
 For more information about prerequisites, credentials management, or permissions for vSphere, visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/getting_started.md).
 
 ```bash
-kubectl create -f https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/download/v0.5.0/infrastructure-components.yaml
+kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-vsphere" asset:"infrastructure-components.yaml" version:"0.5.x"}}
```

Comment:
now we just need a prepreprocessor to process the right version numbers before the preprocessor runs 😆